lib/livebook/runtime/evaluator.ex

```diff
@@ -620,6 +620,73 @@ defmodule Livebook.Runtime.Evaluator do
     end
   end
 
+  defp eval("erlang", code, binding, env) do
+    try do
+      erl_binding =
+        binding
+        |> List.foldl(:erl_eval.new_bindings(), fn {name, value}, erl_binding ->
+          :erl_eval.add_binding(
+            name
+            |> :erlang.atom_to_binary()
+            |> Macro.camelize()
+            |> :erlang.binary_to_atom(),
+            value,
+            erl_binding
+          )
+        end)
+
+      {:ok, tokens, _} = :erl_scan.string(code |> String.to_charlist())
+      {:ok, parsed} = :erl_parse.parse_exprs(tokens)
```

Comment:
I know this is a POC but we will have to provide better error handling here. :)

Reply:
Well, in my defense, it is guarded by a `try` block :)

Would you like to see a `SyntaxError` or something like that raised from the Erlang counterpart?

Reply:
I am almost sure we convert this into a diagnostic for Elixir so we can show proper line and message directly in the editor.

Reply:
And in the future we will extract warnings to show in the editor too!